nodejs/Dockerfile-kali

```diff
@@ -0,0 +1,21 @@
+# A docker image for the func container.
+
+# debian variant is the official standard node image (larger than the alpine image)
+FROM kalilinux/kali-rolling
```

Comment:
I could not find tags for specific versions but rolling seems like latest - would have compatibility issues?

Reply:
Yea, kept it at latest for now. We'll have to reply on users to report when and if they run into an issue